### PR TITLE
Changelog 1.2

### PR DESCRIPTION
**Pull request by ah1053**

### CODE REVIEW

File: EdgeMultiplay/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
 # Changelog
 All notable changes to this EdgeMultiplay Unity Client will be documented in this file.
 
+## [1.2.0] - 2021-05-05
+
+### Fix & Improvements:
```
Comment (vmohanx) [inline EdgeMultiplay/CHANGELOG.md:6]: minor : Fix**es**

File: EdgeMultiplay/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+- Fixed customizable fallback location for Oculus/Non Phone devices in your GameManager.cs
+```
+EdgeManager.integration.useFallbackLocation = true;
+EdgeManager.integration.setFallbackLocation(longitude, latitude);
+ConnectToEdge(useFallBackLocation:true);
+```
+- Use SendGamePlayEvent() and SendGamePlayEventUDP() in your PlayerManager to SendGamePlayEvents.
+
+### New Features:
+- Server Stats, Now you can see the live server stats from EdgeMultiplay Menu in Unity Editor.
```
Comment (vmohanx) [inline EdgeMultiplay/CHANGELOG.md:17]: slight re-write : "Server Stats : View analytics of an Edge Multiplay instance in a web browser. To view the closest instance's stats, click on the menu button Server Stats under the EdgeMultiplay menu in the Unity Editor"

File: EdgeMultiplay/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+- New Ownership Transfer options : TakeOver and RequestOwnership.
```
Comment (vmohanx) [inline EdgeMultiplay/CHANGELOG.md:18]: New "Orphan Observable"

**Review by vmohanx — APPROVED** [review]
Minor changes but looks good

File: EdgeMultiplay/CHANGELOG.md (hunk 3 above)
Comment (ah1053) [reply EdgeMultiplay/CHANGELOG.md:18]: Screen Shot 2021-05-05 at 4 05 45 PM

Comment (ah1053) [reply EdgeMultiplay/CHANGELOG.md:18]: We had Orphan Observable in the previous release.

**Pull request merged**